test/test_browser.py

```diff
@@ -3432,6 +3442,8 @@ def test_sdl2_mixer_music(self, formats, flags, music_name):
       '-sUSE_SDL=2',
       '-sUSE_SDL_MIXER=2',
       '-sSDL2_MIXER_FORMATS=' + ','.join(formats),
+      '-lc++',
+      '-lc++abi',
```

Comment:
Why are these needed?

Reply:
Because libmodplug is written in C++.  I've added a comment and limited it to just that mode.